components/GameCard.jsx

```diff
@@ -13,7 +13,7 @@ export default function GameCard({ game }) {
 const StyledLink = styled(Link)`
   height: 10rem;
   min-width: 100px;
-  background-color: var(--backgroundSubSection);
+  background-color: ${(props) => props.color};
```

Comment:
styled component props need to be prefixed with a `$`, otherwise they will be forwarded to the html elements and they will complain about unknown attributes like "color" (which are not defined for plain html elements)